src/components/users/LoginForm.vue

```diff
@@ -129,17 +195,37 @@ export default {
       provider.addScope('https://www.googleapis.com/auth/calendar.events')
       provider.addScope('https://www.googleapis.com/auth/userinfo.email')
       try {
-        await firebase.auth().signInWithPopup(provider)
+        const createUser = await firebase.auth().signInWithPopup(provider)
+        const result = await createUser
+        // await firebase
+        //   .firestore()
+        //   .collection("user_data")
+        //   .add({ uid: result.user.uid });
+        sessionStorage.setItem('userId', result.user.uid)
```

Comment:
actually I use it because after google login and then save data according I need with which uid actually logged in then I save related information under that